nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/login-identity-providers.xml

```diff
@@ -50,7 +50,7 @@
         'Connect Timeout' - Duration of connect timeout. (i.e. 10 secs).
         'Read Timeout' - Duration of read timeout. (i.e. 10 secs).
        
-        'Url' - Url of the LDAP servier (i.e. ldap://<hostname>:<port>).
+        'Url' - Comma separated list of URLs of the LDAP server (i.e. ldap://<hostname>:<port>).
```

Comment:
I think this documentation update should probably be reflected in administration-guide.adoc too.